From e8bc50eecc2b2b3b2d66793e85f9702f2c916296 Mon Sep 17 00:00:00 2001
From: Rashed Abdel-Tawab <rashed@linux.com>
Date: Fri, 17 Aug 2018 09:20:05 -0700
Subject: [PATCH 05/15] overlay/common: Unlock swipe gesture nav bar

 * The setting will only appear on devices with software nav bar.

Change-Id: I243992335282456aa8198228382094ddeb5e577b
---
 overlay/common/frameworks/base/core/res/res/values/config.xml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/overlay/common/frameworks/base/core/res/res/values/config.xml b/overlay/common/frameworks/base/core/res/res/values/config.xml
index 9e5cb3a1..afcd8e0d 100644
--- a/overlay/common/frameworks/base/core/res/res/values/config.xml
+++ b/overlay/common/frameworks/base/core/res/res/values/config.xml
@@ -92,4 +92,7 @@
 
     <!-- Flag indicating whether round icons should be parsed from the application manifest. -->
     <bool name="config_useRoundIcon">true</bool>
+
+    <!-- Whether or not swipe up gesture's opt-in setting is available on this device -->
+    <bool name="config_swipe_up_gesture_setting_available">true</bool>
 </resources>
-- 
2.17.1

